app/src/main/java/com/codefororlando/petadoption/feed/PetFeedViewPagerAdapter.kt

```diff
@@ -0,0 +1,45 @@
+package com.codefororlando.petadoption.feed
+
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentPagerAdapter
+import android.view.ViewGroup
+import com.codefororlando.petadoption.feed.base.PetFeedView
+import com.codefororlando.petadoption.feed.popular.PopularPetFeedFragment
+import com.codefororlando.petadoption.feed.watching.WatchingPetFeedFragment
+
+class PetFeedViewPagerAdapter(fm: FragmentManager?) : FragmentPagerAdapter(fm) {
+
+    private val POPULAR_FEED_INDEX = 0
+    private val FOLLOWING_FEED_INDEX = 1
+    private val SIZE = 2
+
+    var mCurrentItem: PetFeedView? = null
```

Comment:
You don't need to hold the value locally. Callers to the adapter should be able to get the current position from the pager itself then ask the adapter to return the item at that position.

Pager method for the current item.
https://developer.android.com/reference/android/support/v4/view/ViewPager.html#getCurrentItem()